Week1/assignments/pembe-miriam/Staircase.java

```diff
@@ -0,0 +1,18 @@
+public class Staircase {
+
+  public static void main(String[] args) {
+    int number = 4;
+
+    for (int i = 1; i <= number; i++) {
+      for (int k = number; k > i; k--) {
+        System.out.print(" ");
+      }
```

Comment:
This works.
Can you figure out the time complexity of this algorithm and is there any way to make it better?